third_party/terraform/data_sources/data_source_google_kms_secret.go

```diff
@@ -1,26 +1,35 @@
 package google
 
 import (
-	"google.golang.org/api/cloudkms/v1"
-
+	"context"
 	"encoding/base64"
 	"fmt"
-	"github.com/hashicorp/terraform/helper/schema"
 	"log"
 	"time"
+
+	"github.com/hashicorp/terraform/helper/schema"
+	kmspb "google.golang.org/genproto/googleapis/cloud/kms/v1"
 )
 
 func dataSourceGoogleKmsSecret() *schema.Resource {
 	return &schema.Resource{
 		Read: dataSourceGoogleKmsSecretRead,
 		Schema: map[string]*schema.Schema{
 			"crypto_key": {
-				Type:     schema.TypeString,
+				Type: schema.TypeString,
+				Description: "Full ID of the crypto key to use for decryption in the format" +
```

Comment:
nit: We don't use these descriptions anywhere else, so you can remove them. Or is there a strong reason we should we start using them?

Reply:
Sorry - I added those because I thought the website pulled them for autogen, but I was thinking about something different. Removed.